README.md

```diff
@@ -150,11 +150,12 @@ The following table lists the global, default, and other parameters supported by
 | -------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
 | `global.enterprise` | Enable the enterprise features | `true` |
 | `global.hosted` | Enabled Hosted Mender specific features | `false` |
+| `global.priorityClassName` | Optional pre-existing priorityClassName to be assigned to the workwloads | `nil` |
 | `global.image.registry` | Global Docker image registry | `registry.mender.io` |
 | `global.image.username` | Global Docker image registry username | `nil` |
 | `global.image.password` | Global Docker image registry username | `password` |
 | `global.image.tag` | Global Docker image registry tag | `mender-3.6.2` |
-| `global.imagePullSecrets` | Global Docker registry secret names as an array | `[]` (does not add image pull secrets to deployed pods)  |
```

Comment:
`global.imagePullSecrets` is currently not in use

Reply:
Correction: is used by subcharts (e.g.: Redis). I'd leave this out entirely to avoid mistakes